frontend/src/components/ui/navigation/navigation-button.ts

```diff
@@ -66,7 +66,7 @@ export class Button extends TailwindElement {
     return html`<${tag}
       type=${this.type === "submit" ? "submit" : "button"}
       class=${[
-        tw`flex w-full cursor-pointer items-center gap-2 rounded font-medium leading-[16px] outline-primary-600 transition hover:transition-none focus-visible:outline focus-visible:outline-3 focus-visible:outline-offset-1 disabled:cursor-not-allowed disabled:bg-transparent disabled:opacity-50`,
+        tw`flex min-h-8 w-full cursor-pointer items-center gap-2 rounded font-medium leading-[16px] outline-primary-600 transition hover:transition-none focus-visible:outline focus-visible:outline-3 focus-visible:outline-offset-1 disabled:cursor-not-allowed disabled:bg-transparent disabled:opacity-50`,
```

Comment:
I thiiiiink this messes up the instances where this should be smaller — lemme get back to you tomorrow or monday and properly address this

Reply:
I tried to address with padding initially but it added the padding to the outside of the component?  Either way, not convinced it should be any smaller than 32px tall?  Is there a situation I'm missing where that should be true?

Reply:
Yes, pagination buttons are 24px tall

Reply:
Ahh, good catch!!

Reply:
Going to remove that from this PR, lets deal with it later and keep the scope small (just icon changes) for this one?